R/execution.r

```diff
@@ -126,7 +147,12 @@ execute = function(request) {
     err <<- list()
     
     handle_error <- function(e) {
-        err <<- list(ename = 'ERROR', evalue = toString(e), traceback = list(toString(e)))
+        calls <- head(sys.calls()[-seq_len(23)], -3)
```

Comment:
yes. they remove everything that ends up in the call stack

1. by us and evaluate (23 levels deep)
2. by the error handler (3 levels on the other end)